### DISCUSSION TITLE
Why is .github/wiki the default?

### DISCUSSION THREAD

**Discussion by jcbhmr** [spenserblack/actions-wiki#31] [Q&A] [answered]
Genuinely curious; why not `wiki/` or `docs/`, why hidden in `.github/`?

**Comment by spenserblack** [comment, marked as answer]
Since it's not part of "main development", and it's related to a GitHub feature, I figured it should go in `.github`. Like `.github/dependabot.yml` and `.github/workflows/*`, not really your project, but configuration for the host of your project. Also, if your project was mirrored between GitHub and GitLab, I could imagine a `.gitlab/wiki` and a `.github/wiki`, since, as competitors, there's not guarantee they'd share the same wiki repository structure.

This was also partially inspired by the [repository settings app](https://github.com/repository-settings/app). Specifically, the part where you place file(s) in `.github/` that allow outside contributors to contribute to and change parts otherwise inaccessible to non-maintainers.
